specifyweb/frontend/js_src/lib/components/QueryBuilder/Results.tsx

```diff
@@ -40,17 +24,18 @@ import {
 } from '../Permissions/helpers';
 import { fetchPickList } from '../PickLists/fetch';
 import { generateMappingPathPreview } from '../WbPlanView/mappingPreview';
-import { RecordSetCreated, recordSetFromQueryLoading } from './Components';
+import { CreateRecordSet } from './CreateRecordSet';
 import type { QueryFieldSpec } from './fieldSpec';
 import type { QueryField } from './helpers';
 import { sortTypes } from './helpers';
+import { useFetchQueryResults } from './hooks';
 import { QueryResultsTable } from './ResultsTable';
 import { QueryToForms } from './ToForms';
 import { QueryToMap } from './ToMap';
 
 export type QueryResultRow = RA<number | string | null>;
 
-type Props = {
+export type Props = {
```

Comment:
if you make something globally available, make the name more verbose (`QueryResultsProps`) to prevent name collisions (confuses auto-importer in the editor)